frontend/packages/shared/src/api/mutations.ts

```diff
@@ -67,7 +68,7 @@ export const addAppAttachmentMetadata = (org: string, app: string, payload: Appl
 export const addLanguageCode = (org: string, app: string, language: string, payload: AddLanguagePayload) => post<void, AddLanguagePayload>(textResourcesPath(org, app, language), payload);
 export const addLayoutSet = (org: string, app: string, layoutSetIdToUpdate: string, payload: LayoutSetConfig) => post<AddLayoutSetResponse>(layoutSetPath(org, app, layoutSetIdToUpdate), payload);
 export const deleteLayoutSet = (org: string, app: string, layoutSetIdToUpdate: string) => del(layoutSetPath(org, app, layoutSetIdToUpdate));
-export const updateLayoutSet = (org: string, app: string, layoutSetIdToUpdate: string, payload: LayoutSetConfig) => put(layoutSetPath(org, app, layoutSetIdToUpdate), payload);
+export const updateLayoutSet = (org: string, app: string, layoutSetIdToUpdate: string, newLayoutSetId: string) => put(layoutSetPath(org, app, layoutSetIdToUpdate), JSON.stringify(newLayoutSetId), { headers: { 'Content-Type': 'application/json' } });
```

Comment:
Why do we need to stringify a string? 😄

Reply:
Wops, testing phase that was not cleaned up. Nice catch. Removed 🤝🏼